packages/expo-web-browser/src/WebBrowser.ts

```diff
@@ -212,6 +212,10 @@ export function dismissBrowser(): void {
  * # On iOS:
  * Opens the url with Safari in a modal using `ASWebAuthenticationSession`. The user will be asked
  * whether to allow the app to authenticate using the given url.
+ * To handle redirection back to the mobile application, the redirect URI set in the authentication server 
+ * has to use the protocol provided as the scheme in `app.json` [expo.scheme](../../config/app/#scheme) 
```

Comment:
```suggestion
 * has to use the protocol provided as the scheme in **app.json** [`expo.scheme`](../../config/app/#scheme) 
```

As per our writing style guide, [filenames have to be bold](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#filenames-as-bold-text).